src/oscar/templates/oscar/dashboard/layout.html

```diff
@@ -19,12 +19,10 @@
 {% block extrastyles %}
     {{ block.super }}
     {% compress css %}
-        {% block jquery-ui-css %}
-            <link rel="stylesheet" href="{% static "oscar/js/jquery-ui/jquery-ui.css" %}">
-        {% endblock %}
         <link rel="stylesheet" href="{% static "oscar/js/select2/select2.css" %}" />
         <link rel="stylesheet" href="{% static "oscar/css/select2-bootstrap.css" %}" />
         <link rel="stylesheet" href="{% static "oscar/js/bootstrap-datetimepicker/bootstrap-datetimepicker.css" %}" />
+        <link rel="stylesheet" type="text/css" href="{% static "oscar/css/datetimepicker.css" %}" />
```

Comment:
The three lines above have no type attribute. So we should either add them there too or remove it from the new line